builder/builder.mk

```diff
@@ -793,6 +793,8 @@ release/promote-oss/dev-to-rc:
 		$(MAKE) VERSION_OVERRIDE=$${veroverride} push-manifests  ; \
 		$(MAKE) VERSION_OVERRIDE=$${veroverride} publish-docs-yaml ; \
 		$(MAKE) clean-manifests ; \
+		gavers=$$(echo $(RELEASE_VERSION) | sed 's/-rc.*//'); \
```

Comment:
you can also use $(VERSIONS_YAML_VERSION), and it's actually more reliable for -ea releases